Makefile

```diff
@@ -32,7 +32,11 @@ RPM_VERSION ?= $(DEB_VERSION)
 RPM_REVISION ?= 0
 
 # used by hack/jenkins/release_build_and_upload.sh and KVM_BUILD_IMAGE, see also BUILD_IMAGE below
-GO_VERSION ?= 1.15.8
+GO_VERSION ?= 1.16.0
```

Comment:
please change this version everywhere in the code (jenkins scripts and gh action things)

Reply:
done